R/generateTPresids.R

```diff
@@ -23,7 +23,7 @@
 #' @export
 generate.TP.resids <- function(emulator, ngen, method = 1){
 
-    Ngrid <- ncol(emulator$meanfldT$r)
+    Ngrid <- nrow(emulator$meanfldT$w)
 
```

Comment:
Why did this change?

Reply:
I had to update it so that it would get the number of grid cells from something that would even be included in the reduced emulator. Since you don't technically need the input residuals stored in `$r` to generate new fields, those matrices got dropped.